bydbctl/internal/cmd/stream.go

```diff
@@ -0,0 +1,322 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/apache/skywalking-banyandb/pkg/version"
+	"github.com/ghodss/yaml"
+	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func newStreamCmd() *cobra.Command {
+	StreamCmd := &cobra.Command{
+		Use:     "stream",
+		Version: version.Build(),
+		Short:   "banyandb stream schema related Operation",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			return cmd.Parent().PersistentPreRunE(cmd.Parent(), args)
+		},
+	}
+
+	StreamCreateCmd := &cobra.Command{
+		Use:     "create",
+		Version: version.Build(),
+		Short:   "banyandb stream schema create Operation",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) (err error) {
+			return cmd.Parent().PersistentPreRunE(cmd.Parent(), args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			client := resty.New()
+			addr, err := cmd.Flags().GetString("addr")
```

Comment:
Please use `Addr` and `JSON` as https://github.com/spf13/cobra/blob/main/user_guide.md#working-with-flags